lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -16,11 +16,13 @@ defmodule ExUnitTest do
       test "false" do
         assert false
       end
+
+      test "not implemented test"
```

Comment:
Can we define this in a separate place and also check the output printed? Thank you.